spec/jobs/clean_up_unused_staged_users.rb

```diff
@@ -0,0 +1,37 @@
+require 'rails_helper'
+
+RSpec.describe Jobs::CleanUpUnusedStagedUsers do
+  let(:user) { Fabricate(:user) }
+  let(:staged_user) { Fabricate(:user, staged: true) }
+
+  context 'when staged user is unused' do
+    it 'should clean up the staged user' do
+      user
```

Comment:
What are you calling `user` here? Can't you use `let!(:user) { Fabricate(:user) }`?